apps/mobile/src/navigations/root/root.tsx

```diff
@@ -8,7 +8,7 @@ import { RootScreenName } from "~/libs/enums/enums";
 import { useAppSelector } from "~/libs/hooks/hooks";
 import { type RootNavigationParameterList } from "~/libs/types/types";
 import { Auth } from "~/screens/auth/auth";
-import { WelcomeScreen } from "~/screens/welcome-screen/welcome-screen";
+import { QuizEntryScreen } from "~/screens/quiz-entry-screen/quiz-entry-screen";
```

Comment:
```suggestion
import { QuizEntry } from "~/screens/quiz-entry/quiz-entry";
```
It is already in screens folder, so we know, that it is a screen